Introduce Buildkit to AutoBuilds (#10972)
docker/docker.github.io: docker-hub/builds/index.md

`/(alice|bob)-v([0-9.]+)/` -->

> ### Build image with Buildkit

Reviewer: -  s/Buildkit/BuildKit/ ? (capital `K`)
- Should it be plural (images)?

```suggestion
### Build images with BuildKit
```

> You can enable the Buildkit builder by setting the environment variable `DOCKER_BUILDKIT=1`([enviroment variables](index.md#environment-variables-for-builds)) in the Configure automated build section ([configure builds](index.md#configure-automated-build-settings)).

Reviewer: - Given it's the same page, we don't strictly need the `index.md` (and would work around some bugs in Jekyll/plugins).
- I think we could just "Configure automated build section" as the caption for the link here
- s/Buildkit/BuildKit/ ? (capital `K`)
- Wrap to 80 chars 🤗 

```suggestion
You can enable the BuildKit builder by setting the `DOCKER_BUILDKIT=1`
[environment variable](#environment-variables-for-builds) in the
[Configure automated build settings](#configure-automated-build-settings) section.
```

> For more information on Buildkit please see [buildkit](https://docs.docker.com/develop/develop-images/build_enhancements/).

Reviewer: Try to avoid "please"; also, we should avoid using a hard-coded URL for docs.docker.com (so that preview builds of the docs link correctly), perhaps;

```suggestion
Refer to the [build images with BuildKit](../../develop/develop-images/build_enhancements.md)
page for more information on BuildKit.
```
